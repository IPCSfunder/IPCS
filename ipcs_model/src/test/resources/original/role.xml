<?xml version='1.0' encoding='UTF-8'?>
<dataset>
	<permission PERMISSION_OBJID="1" NAME = "ADD_UPDATE_REMOVE_TEACHER"/>
	<role role_objid = "1" name = "ADMIN"/>
	<role role_objid = "2" name = "CHILDREN"/>
	<role role_objid = "3" name = "STAFF"/>
	<role_permission></role_permission>
</dataset>