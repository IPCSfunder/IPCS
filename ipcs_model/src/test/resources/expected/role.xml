<?xml version='1.0' encoding='UTF-8'?>
<dataset>
    <role name = "ADMIN"/>
    <role name = "CHILDREN"/>
    <role name = "STAFF"/>
    <role name="PARENT"/>

    <permission PERMISSION_OBJID="1" NAME = "ADD_UPDATE_REMOVE_TEACHER"/>
    <permission PERMISSION_OBJID="2" NAME = "ADD_UPDATE_REMOVE_ACTIVITY"/>

    <role_permission permission_fk="2"></role_permission>
    <role_permission permission_fk="1"></role_permission>
    <role_permission permission_fk="2"></role_permission>
</dataset>